daemons/dss-notify-v2/Readme.md

```diff
@@ -30,8 +30,8 @@ Bundle creation notifications have the format
 
 ```
 {
-  'dss_api': {dss_api_url}
-  'bundle_url': {dss_api}"/v1/bundles/{uuid}?version={version}
+  'dss_api': {dss_api_url},
```

Comment:
```
'dss_api': "{dss_api_url}",
'bundle_url': "{dss_api_url}/v1/bundles/{uuid}?version={version}",
```

Reply:
from what i see the other parts of this readme, strings are double quote wrapped. variables are {} wrapped.

Reply:
The two main points:

`dss_api_url` and `dss_api` are the same thing, so should have the same name.

`{dss_api}"/v1/bundles/{uuid}?version={version}"` doesn't make any sense.

Reply:
updated.

Reply:
updated.